templates/emails/com/onboarding/welcome-proton-user.html

```diff
@@ -1,32 +1,50 @@
 {% extends "base.html" %}
 
 
-{% block logo %}<img src="{{URL}}/static/logo-sl-proton.svg" style="width: 150px; margin: auto">{% endblock %}
+{% block logo %}<img src="{{ URL }}/static/logo-sl-proton.svg" style="width: 150px; margin: auto">{% endblock %}
 
 {% block content %}
   {% call text() %}
     Welcome to SimpleLogin, a service developed by Proton to protect your email address!
   {% endcall %}
 
   {% call text() %}
-    This is the first email you receive via your first alias {{ to_address }}
+    This is the first email you receive via your <b>first alias</b> {{ to_address }}
   {% endcall %}
 
   {% call text() %}
-    This alias was automatically created the first time you accessed SimpleLogin.
+    This alias is automatically created when you use SimpleLogin for the first time.
     Emails sent to it are forwarded to your Proton mailbox.
-    If you want to reply to an email, just hit "Reply" and the response will come from this alias.
-    Your personal email address will stay hidden.
+    If you want to reply to an email, just hit "Reply" and the response will come from your alias.
+    Your personal email address stays hidden.
```

Comment:
Nits:
```suggestion
    Your personal email address remains hidden.
```

Reply:
this wording is corrected by Andy so I wouldn't change it :)

Reply:
Ok :)